lib/tasks/one-off/doc_sentencing_counties.rake

```diff
@@ -0,0 +1,38 @@
+require 'open-uri'
+
+namespace :doc do
+  desc 'Pull in attorneys from OK Bar site'
+  task :sentencing_counties do
+    county_court = ActiveRecord::Base.connection.execute("SELECT DISTINCT sentencing_county from doc_sentences where sentencing_county ilike '%county court%' order by sentencing_county")
+
+    county_court = county_court.column_values(0).sort
+    sent_counties = ::Doc::Sentence.distinct.pluck(:sentencing_county).sort
```

Comment:
I'd consider renaming  sent_counties to `sentencing_county_names` (it's longer but clearer what you're working with...if it's an object vs a string)
and also in your loop renaming to `sentencing_county_name`